### ISSUE TITLE
Add a "Tips" section

### ISSUE THREAD

**Issue by Laegel** [tauri-apps/tauri-docs#73]
Create this section in the docs (Usage > Tips).
This section is supposed to bring useful clues to consumers so they can learn how to deal with some edge-cases, for example.

Add the following items:

- 
> And a word of advice: don't rely on remote resources. That will make your app boot slower and if either the user or remote service are offline then infinitely slower.
It's one of the key differences between an app and a website.
Also remote assets are always a vulnerability.
- Make sure to make your app cross-browser compatible since Webview relies on different browser views;
- https://github.com/tauri-apps/vue-cli-plugin-tauri/issues/14#issuecomment-651377787;

**Comment by simonhyll** [comment]
I don't think we are gonna do a "Tips" section in the new version of the docs, at least not quite like this. Best practises like this are gonna be a bit more scattered about on the site where they feel relevant to bring up.
I'll track it though for 2.0 so the information you brought up isn't forgotten to be brought up somewhere.
